examples/advanced/sklearn-kmeans/kmeans_job_clientapi.py

```diff
@@ -135,7 +135,7 @@ def main():
             script_args=f"--data_path {data_path} "
             f"--train_start {train_start} --train_end {train_end} "
             f"--valid_start {valid_start} --valid_end {valid_end}",
-            params_exchange_format="raw",
+            framework="raw",
```

Comment:
Is this example not using "numpy"?

Reply:
based on this "exchange_format" seems it is not, @ZiyueXu77 can confirm

Reply:
numpy may also work, I just put raw to make sure no further process needed to modify the content for this